Include/Minimal/JcfGlobal.inc

```diff
@@ -0,0 +1,3 @@
+{$I jedi.inc}
+{.$DEFINE JCF_GUI}
```

Comment:
I do not the the idea of having this file. moreover with the "commented" conditionals

Reply:
I integrated your formatting code in my app, without GUI (which uses JvInterpreter to run scripts).
I don't want my app to be able to read/write to the registry.

Thus, I tried to stick to the bare minimum of formatting code to be added into my app.
It is a good idea to separate the GUI from the formatting logic, but it was too much work to make a complete separation and the use of these IFDEFs was the fastest and easiest solution for that requirement.